modules/eventing/pages/eventing-Terminologies.adoc

```diff
@@ -110,6 +110,8 @@ NOTE: You can have multiple Eventing Functions running different code listening
 
 When a source collection is deleted, all deployed (or paused) Eventing Functions associated with this source collection are undeployed.
 
+The Listen To Location as of 7.1.1 can listen to multiple collections via a wildcard of "+++*+++" for the scope and/or the collection. For these Functions if the bucket binding used by the JavaScript code also contains a wild card of "+++*+++" for the scope and/or the collection only the Advanced Keyspace Accessors will be able to read or write the Data Service (or KV). [.status]#Couchbase Server 7.1.1#
```

Comment:
I would be more inclined to use backticks rather than quotes for the wildcards:
```suggestion
The Listen To Location as of 7.1.1 can listen to multiple collections via a wildcard of `+++*+++` for the scope and/or the collection. 
For these Functions if the bucket binding used by the JavaScript code also contains a wild card of `+++*+++` for the scope and/or the collection only the Advanced Keyspace Accessors will be able to read or write the Data Service (or KV). [.status]#Couchbase Server 7.1.1#
```